### PR TITLE
Avoid fetching the name space if possible when start SSM server

### PR DESCRIPTION
**Pull request by huafengw**

### CODE REVIEW

**Comment by littlezhou** [comment]
@huafengw  table changed

**Comment by huafengw** [comment]
@littlezhou

File: smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/metric/fetcher/InotifyEventFetcher.java [hunk 1]
```diff
@@ -64,41 +68,92 @@ public InotifyEventFetcher(DFSClient client, MetaStore metaStore,
...
+  static boolean canContinueFromLastTxid(DFSClient client, Long lastId) {
+    try {
+      if (client.getNamenode().getCurrentEditLogTxid() == lastId) {
+        return true;
+      }
+      DFSInotifyEventInputStream is = client.getInotifyEventStream(lastId);
+      EventBatch eventBatch = is.poll();
+      return eventBatch != null;
+    } catch (Exception e) {
+      e.printStackTrace();
```
Comment (littlezhou) [inline smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/metric/fetcher/InotifyEventFetcher.java]: I don't know whether "e.printStackTrace()" can output the info into log file or not, but maybe better to use LOG instead as it provides more flexibility and looks more formal. Thanks!

**Comment by littlezhou** [comment]
LGTM

**Pull request merged**